cmd/ocsp-updater/main.go

```diff
@@ -422,6 +440,13 @@ func (updater *OCSPUpdater) oldOCSPResponsesTick(ctx context.Context, batchSize
 		return err
 	}
 
+	for _, s := range statuses {
```

Comment:
@jsha this query joins on `certificates` presently to check `cert.expires > now()`. If I remove the JOIN and look at the `cs.isExpired` field for this purpose I would have to do a separate `SELECT` per certificate status to check cert `expires` vs the certStatus `isExpired` no?